docs/DeveloperGuide.md

```diff
@@ -394,13 +393,37 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 2a1. System displays error message and does not clear command line.
     * 2a2. User enters command with new index.
 * Steps 2a1-2a2 are repeated until index entered is correct.
-* Use cases resumes from step 3.
+* Use case resumes from step 3.
 
 * 2b. System detects error in format of entered command.
     * 2b1. System displays error message and does not clear command line.
     * 2b2. User enters new command.
 * Steps 2b1-2b2 are repeated until all details entered are correct.
-* Use cases resumes from step 3.
+* Use case resumes from step 3.
+
+**Use case: UC06 - Receiving tuition fee from a student**
+
+**MSS**
+
+1. User requests to <ins>find a student(UC05)</ins>.
+2. User enters command to record payment received from the specified student after a lesson.
+3. System calculates the tuition fee paid by the student for the lesson.
+4. System updates the total tuition fee paid by the student.
+5. System displays success message.
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. System cannot find the specified student.
+    * 1a1. User adds the student to the system (UC01).
```

Comment:
should this be <ins> too?

Reply:
Thanks for catching this! Ive added in the <ins>